src/OrchardCore.Modules/OrchardCore.OpenId/Startup.cs

```diff
@@ -138,4 +139,32 @@ public override void ConfigureServices(IServiceCollection services)
             });
         }
     }
+
+    [Feature(OpenIdConstants.Features.Client)]
+    public class ClientStartup : StartupBase
+    {
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSecurity();
+
+            services.AddScoped<IPermissionProvider, Permissions>();
+            services.AddScoped<INavigationProvider, AdminMenu>();
+            services.AddScoped<IDisplayDriver<ISite>, OpenIdConnectSettingsDisplayDriver>();
+            services.AddSingleton<IOpenIdClientService, OpenIdClientService>();
+            services.AddAuthentication().AddCookie();
```

Comment:
Can it conflict with other calls to `AddAuthentication`? Or should it just depend on the feature that does it?

Reply:
It should be removed since it's not even used.